commerce-service/service.xml

```diff
@@ -284,6 +284,7 @@
 		<column name="orderStatus" type="int" />
 		<column name="printedNote" type="String" />
 		<column name="requestedDeliveryDate" type="Date" />
+		<column name="manualAdjusted" type="boolean" />
```

Comment:
Hey Alessio, can we call this `manuallyAdjusted` instead? That sounds more correct to me.

Reply:
Yes, I will change it. Thanks